lib/execAsync.js

```diff
@@ -1,6 +1,7 @@
 const exec = require('child_process').exec;
 
 module.exports = function (command, options) {
+    options = Object.assign({}, options, { maxBuffer: (1024 * 1024 * 10) });
```

Comment:
A comment would be nice I think

Reply:
I think this should also be set as a command line parameter as it may make more sense for some to limit it to 1-2MB whereas others it should be 10-50MB.